app/models/group.rb

```diff
@@ -3,12 +3,13 @@ class Group < ApplicationRecord
   has_many :users, through: :user_groups
   has_many :lists
   has_many :invitations
-  belongs_to :user, foreign_key: :owner_id
-  validates :gift_due_date, presence: true
-  belongs_to :owner, foreign_key: :owner_id, class_name: 'User'
-  validates_uniqueness_of :name
+  validates :name, presence: true
+  validates_uniqueness_of :name, message: 'Choose a different group name, this name is already in use.'
```

Comment:
Nice message here, didn't know you could set that with the rails validation helper

Reply:
Thanks. I fixed this exact message in PR #23 because the message prepends the name of the column before the custom message.